toolbox/load_agol_layer/src/settings.js

```diff
@@ -53,6 +53,13 @@ const LAYERS = {
     fields: ["ENGINEERING_AREA_ID", "ATD_ENGINEER_AREAS"],
     tableName: "engineering_areas",
   },
+  jurisdictions: {
+    service_name: "BOUNDARIES_jurisdictions_planning",
+    layer_id: 0,
+    query_params: { ...DEFAULT_ESRI_QUERY_PARAMS, where: "JURISDICTION_LABEL = 'AUSTIN FULL PURPOSE'" },
```

Comment:
we're filtering out all features that are not AFP. there are thousands of features in this layer, and a single jurisdiction label can have multiple rows.  E.g. there are currently two `AUSTIN FULL PURPOSE` features which, when combined, represent the entire AFP.

Reply:
thanks for pointing this out because I was curious about the 2 rows in the DB. Makes sense! 🙏